src/Features/Core/Portable/CodeFixes/AddExplicitCast/AbstractAddExplicitCastCodeFixProvider.cs

```diff
@@ -175,7 +175,7 @@ private static void ReportTelemetryIfNecessary(ImmutableArray<(TExpressionSyntax
             var root = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             var spanNodes = diagnostics.SelectAsArray(
                 d => root.FindNode(d.Location.SourceSpan, getInnermostNodeForTie: true)
-                         .GetAncestorsOrThis<TExpressionSyntax>().FirstOrDefault());
+                         .GetAncestorsOrThis<TExpressionSyntax>().First());
```

Comment:
The code looks very confidential about it (defensive in outer code but not this), so I changed to `First`.